tutor/src/screens/assignment-edit/delete-btn.js

```diff
@@ -0,0 +1,90 @@
+import { React, PropTypes, styled, observer, observable, action } from 'vendor';
+import { AsyncButton } from 'shared';
+import { Icon } from 'shared';
+import TourAnchor from '../../components/tours/anchor';
+import { Modal, Button } from 'react-bootstrap';
+
+const DeleteWrapper = styled(TourAnchor)`
+  margin-left: 1rem;
+`;
+
+// eslint-disable-next-line
+const DeleteModal = observer(({ isVisible, show, onClose, isBusy, onDelete }) => {
+
+  return (
+    <Modal
+      show={show}
+      onHide={onClose}
+      className="settings-delete-assessment-modal">
+      <Modal.Header closeButton={true}>
```

Comment:
I think if you just add `closeButton`, without the `{true}`, the component will treat it as `true`.